backend/src/main/java/se/chalmers/dat265/group1/api/printing/PrintingController.java

```diff
@@ -3,10 +3,15 @@
 import se.chalmers.dat265.group1.api.ApiController;
 import se.chalmers.dat265.group1.model.DataModel;
 import se.chalmers.dat265.group1.model.DigitalPrint;
+import se.chalmers.dat265.group1.model.MagicsData;
+import se.chalmers.dat265.group1.model.StlData;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'se.chalmers.dat265.group1.model.StlData'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)